flow-typed/declarations.js.flow

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
```

Comment:
Extra fancy: add an npm `post-install` to copy this into `installer/flow-typed/`?  No idea if it's common practice, but it seems to mesh with both https://flow.org/en/docs/libdefs/ and https://docs.npmjs.com/misc/scripts#examples

Reply:
I would volunteer to add it later, if it seems worthwhile.  This PR is big enough already.  :)

Reply:
That would be a good idea, is this so that other apps that use react-launch-darkly can make use of the flow declarations?

Reply:
I think so.  The flow doc doesn't specify how folks are meant to acquire the libdefs.

I just noticed that our current installed build includes two empty files where the types were previously defined.  They get transpiled into just `use strict;`

Reply:
postinstall has so many crazy side effects that it should definitely be a separate PR.